install/linux/usr/share/odemis/sim/cryosecom-sim.yaml

```diff
@@ -200,10 +200,12 @@ CryoSECOM: {
   },
   metadata: {
     POS_ACTIVE_RANGE: {'x':  [0.00, 0.003], 'y':  [-0.003, -0.001], 'z':  [0.00, 0.003], } , # stage Z – lens Z, when the lens Z is the closest (highest) from the stage
-    FAV_POS_ACTIVE: {'rx': 0.00, 'rz': 0.0, 'x':  0.0, 'y':  -0.002, 'z':  0.002},
-    FAV_POS_DEACTIVE:  {'rx': 0.00, 'rz': 0.0, 'x':  0.002, 'y':  0.00, 'z':  -0.001},
+    FAV_POS_ACTIVE: {'rx': 0.00, 'x':  0.0, 'y':  -0.002, 'z':  0.002},
+    FAV_POS_DEACTIVE:  {'rx': 0.00, 'rz': 0.1, 'x':  0.002, 'y':  0.00, 'z':  -0.001},
     FAV_AREA:  [-0.001, -0.001, 0.001, 0.001],  # left, top, right, bottom
-    FAV_POS_COATING: {'rx': 0.4363, 'rz': 0.0, 'x':  0.0, 'y':  -0.003, 'z':  0.001},
+    FAV_Z_RANGE: [0.001, 0.003],
```

Comment:
FAV_AREA and FAV_Z_RANGE are now merged into  `POS_ACTIVE_RANGE: {'x':  [-23.e-6, 23.e-6], 'y':  [-33.e-6, 33.e-6], 'z':  [0.001, 0.003], } ,`